tutorials/speech2srt/requirements.txt

```diff
@@ -1,3 +1,3 @@
-google-cloud-speech>=1.2.0
-google-cloud-translate>=2.0.0
-srt>=3.0.0
+google-cloud-speech==1.2.0
+google-cloud-translate==3.0.0
```

Comment:
@lepistom Using a fixed version ensures the changes in API surface doesn't break the existing code.